src/utils/style-utils.js

```diff
@@ -21,6 +24,36 @@ const FONT_FAMILIES = [
   'Verdana, sans-serif',
 ];
 
+export const backgroundSize = {
+  originalSize: 'auto auto',
+  alwaysFit: 'contain',
+  fitWidth: '100% auto',
+  fitHeight: 'auto 100%',
+  fill: '100% 100%',
+  alwaysFill: 'cover',
+};
+
+export const backgroundPosition = {
+  'top-left': 'top left',
+  'center-left': 'center left',
+  'bottom-left': 'bottom left',
+  'top-center': 'top center',
+  'center-center': 'center center',
+  'bottom-center': 'bottom center',
+  'top-right': 'top right',
+  'center-right': 'center right',
+  'bottom-right': 'bottom right',
```

Comment:
@niekvanstaveren I tried with replacing the these values with percentage ones (e.x: 50% 50%) but it didn't work, clicking around position component didn't change the image position so I kept it as it is.